custom_dc/deploy_cloud.md

```diff
@@ -1,39 +1,39 @@
 ---
 layout: default
-title: Deploy a custom instance to Google Cloud
+title: Deploy services to Google Cloud
 nav_order: 7
 parent: Build your own Data Commons
 ---
 
 {:.no_toc}
-# Deploy a custom instance to Google Cloud
+# Deploy services to Google Cloud
 
-This page shows you how to create an artifact and run it in Google Cloud Run. This is step 5 of the [recommended workflow](/custom_dc/index.html#workflow).
+This page shows you how to build a custom services Docker container as a GCP artifact, upload it to the Artifact Registry, and create a Google Cloud Run service. This is step 5 of the [recommended workflow](/custom_dc/index.html#workflow).
 
 * TOC
 {:toc}
 
 ## System overview
 
-When you are ready to launch your custom Data Commons site, we recommend hosting your site in [Google Cloud Run](https://cloud.google.com/run/), which is a serverless solution that is by far the simplest and least expensive option, providing auto-scaling. This is the production setup:
+When you are ready to host your custom Data Commons site in production, you create a [Google Cloud Run](https://cloud.google.com/run/) service for the site. This is the production setup:
 
-![setup4](/assets/images/custom_dc/customdc_setup4.png)
+![services setup](/assets/images/custom_dc/customdc_setup4.png)
 
-You push a locally built Docker image to the [Google Cloud Artifact Registry](https://cloud.google.com/artifact-registry), and then deploy the image in Cloud Run.
+You push a locally built Docker image to the [Google Cloud Artifact Registry](https://cloud.google.com/artifact-registry), and then deploy the image as a Cloud Run service.
 
 ## One-time setup: Create a Google Artifact Registry repository
 
 1. Go to [https://console.cloud.google.com/artifacts](https://console.cloud.google.com/artifacts) for your project.
 1. Click **Create Repository**.
 1. Specify a name for the repository.
 1. In the **Format** option, select **Docker**.
-1. In **Location type**, select **Region**, and specify a region.
+1. In **Location type**, select **Region**, and specify the region you chose for your [Google Cloud SQL instance](/custom_dc/data_cloud.html#location) .
 1. Enable or disable **Immutable image tags** according to the workflow you prefer; that is, if you want to be able to reuse the same Docker tag for new images, keep this option disabled.
 1. Click **Create**.
 
-## Upload the Docker container to Google Cloud
+## Upload the Docker container to the Artifact Registry
 
-This procedure creates a "dev" Docker package that you upload to the Google Cloud Artifact Registry, and then deploy to Google Cloud Run.
+This procedure creates a "dev" Docker package that you upload to the Google Cloud Artifact Registry. Every time you rebuild the image, you need ro rerun this procedure.
```

Comment:
Perhaps - "Any time you rebuild the image and want to deploy it to the cloud, you need to rerun this procedure."

Reply:
Done.